config/webpack/configure-hot-module-replacement.js

```diff
@@ -0,0 +1,45 @@
+var webpack = require('webpack')
+var merge   = require('webpack-merge')
+var path    = require('path')
+
+function configureHotMobuleReplacement(config)
+{
+  let webpackDevServerAddr = process.env['WEBPACK_DEV_SERVER_ADDR'] || 'http://localhost:8080/'
+  config = merge(
+    config,
+    {
+      output: {
+        publicPath: webpackDevServerAddr // necessary for HMR to know where to load the hot update chunks
+      },
+      devServer: {
+        hot: true,
+        contentBase: path.resolve('..', 'public', 'packs'),
+        publicPath: webpackDevServerAddr
+      },
+      plugins: [
+        new webpack.HotModuleReplacementPlugin(),
+        new webpack.NamedModulesPlugin()
```

Comment:
This is not strictly needed for HMR, only to improve the build & console output. I think we should show it in the doc, but let the user add it if he wants. Might also be something to be added to the default Webpacker config with a PR, if the maintainers thinks its a good default